docs/reference/developer_documentation.md

```diff
@@ -105,3 +105,13 @@ poetry publish --build
 1. [Download Node.js](https://nodejs.org/en/download/)
 2. [Install Angular](https://angular.io/guide/setup-local)
 3. See [buildingmotif-app/README.md](buildingmotif-app/README.md)
+
+
+## Running on Docker
+1. Ensure `.env` contains the correct database information. 
+
+2. Up containers with docker-compose.
+    ```
+    docker-compose up --build
+    ```
+3. The api can now be reached at http://127.0.0.1:5000 and the app can be reached but http://localhost:4200.
```

Comment:
is there any reason we're using `127.0.0.1` for one example and `localhost` for the other?

Reply:
🤷‍♀️ I probably just copy and pasted the logs and docs of the tools. I don't know enough about networking to say why they would be differnt.